app/Models/Category.php

```diff
@@ -68,8 +68,11 @@ public function hasFeedsWithError() {
 		return $this->hasFeedsWithError;
 	}
 
-	public function _id($value) {
-		$this->id = $value;
+	public function _id($id) {
+		$this->id = $id;
+		if ($id == FreshRSS_CategoryDAO::DEFAULTCATEGORYID) {
```

Comment:
Approach to set the name of the default category when using it (simplifies the SQL, and makes the i18n possible)

Reply:
Also, I'm not sure it's still possible to rename the default category after that…

Reply:
Right, I will have a look

Reply:
I am looking at another way to implement that. But would you like to allow renaming the default category? I was thinking about disabling this possibility, to simplify and also because it is a special category (a bit like favourites, etc.), but I have no strong opinion.

Reply:
Maybe it's not that important :) feel free to choose what solution you think is the best, I will not be able to review the PR anyway :p

Reply:
Ok, let me then polish/finalise the PR without the possibility to change the name of the default category, and we can re-add this option when you are back if you like (which will be easier than what we have now, just renaming the category after creation). Enjoy your trip!